packages/musescore.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+ 
+class Musescore < Package
+  description "The world's most popular notation app"
+  homepage 'musescore.org'
+  version '4.1.0'
+  license 'GPL-3'
+  compatibility 'aarch64, x86_64'
+  source_url 'https://github.com/musescore/MuseScore/releases/download/v4.1.0/MuseScore-4.1.0.231921359-x86_64.AppImage'
+  source_sha256 'f9a54ad7caf8d2173d67e972ffa68b750bf149f34f4b07d59cbc96dc2c663c79'
+ 
+  depends_on 'p7zip'
+  depends_on 'sommelier'
+  depends_on 'qtbase'
+
+ no_compile_needed
+ 
+  def self.preflight
+   free_space = `echo $(($(stat -f --format="%a*%S" #{CREW_PREFIX})))`.chomp.to_i
+   abort 'Not enough free disk space.  You need at least 317 MB to install.'.lightred if free_space < 317000000
+  end
+  def self.install
+  FileUtils.mv 'squashfs-root/bin' ,"#{CREW_DEST_PREFIX}"
+  FileUtils.mv 'squashfs-root/share' ,"#{CREW_DEST_PREFIX}"
+  FileUtils.mv 'squashfs-root/include' ,"#{CREW_DEST_PREFIX}"
+  FileUtils.mv 'squashfs-root/lib' ,"#{CREW_DEST_PREFIX}"
+  end
```

Comment:
Did you notice a new file located in your local repo at `manifest/x86_64/m/musecore.filelist`?  That will need to be commited in the PR too.

Reply:
i'm creating a bash script https://github.com/Franceschinie/musescore

Reply:
No need to change permissions to executable and definitely don't want to run with sudo.  Installs should add all files inside of `CREW_PREFIX` (`/usr/local` by default) which should be writable for the chronos user.  If this is not the case then the install.sh script needs to be modified to adjust for this relative location.